src/cookbook/networking/fetch-data.md

```diff
@@ -76,7 +76,7 @@ First, create an `Album` class that contains the data from the
 network request. It includes a factory constructor that
 creates an `Album` from JSON.
 
-Converting JSON by hand is only one option.
+Converting JSON using pattern matching is only one option.
```

Comment:
Can you cross link "pattern matching" to https://dart.dev/language/patterns?

Some readers may not be familiar yet.

Reply:
done